setup.py

```diff
@@ -283,7 +283,7 @@ def add_dir_name(dir_name, filenames):
                               '-Ilib/include',
```

Comment:
Could we avoid to modify setup.py in this PR which is already rich enough

Reply:
Sure, do you plan on fixing cpplint as part of flattening? :P